bindings/py/cpp_src/bindings/algorithms/py_Connections.cpp

```diff
@@ -98,49 +98,30 @@ R"(Compatibility Warning: This classes API is unstable and may change without wa
 
     py_Connections.def("computeActivity",
         [](Connections &self, SDR &activePresynapticCells, bool learn=true) {
-            // Allocate buffer to return & make a python destructor object for it.
-            auto activeConnectedSynapses =
-                new std::vector<SynapseIdx>( self.segmentFlatListLength(), 0u );
```

Comment:
Here you've removed the allocator and deallocator for the vector.  So then when is the vector allocated and where is it destroyed?  I suspect the vector will be destroyed when this function exits, leaving the python object with a reference to free'd data.

Reply:
I think what you wanted to do here is to allocate the new vector with a length of zero, and swap the actual data into it.

Reply:
I thought this is ok because computeActivity allocates the vector on stack (not heap, `new`)
https://github.com/htm-community/htm.core/pull/552/files#diff-9fa3fa28527d68cf4a7e360f82508c46R102 

that's why I've removed the destructor. While the other vector is still constructed here in bindings using new, so the destructor is kept.